.github/workflows/semantic-pull-request.yml

```diff
@@ -0,0 +1,17 @@
+name: 'Lint PR'
+
+on:
+  pull_request_target:
```

Comment:
`pull_request_target` will expose token access to actions triggered by a fork. We gotta be very careful and this is what @wlee221 did to mitigate it <- https://github.com/aws-amplify/amplify-ui/pull/855

Reply:
Oh man, you found my fossil 😮

Reply:
This is fine, as long as you don't checkout any of the PR content:

> TL;DR: Combining pull_request_target workflow trigger with an explicit checkout of an untrusted PR is a dangerous practice that may lead to repository compromise. <br /> <br />  https://securitylab.github.com/research/github-actions-preventing-pwn-requests/